src/components/admin/admin-settings-tab.yml

```diff
@@ -2,6 +2,24 @@ adminSettingsTab:
   announcementHeader:
     en: Global announcement
     de: Globale Ankündigung
+  announcementInfo:
+    en: This announcement is shown under the website header on all pages. Users can close it and it will only appear again when the text or type are changed.
+    de: This announcement is shown under the website header on all pages. Users can close it and it will only appear again when the text or type are changed.
+  consentHeader:
+    en: Consent text
+    de: Zustimmungstext
+  homepageDocumentsHeader:
+    en: Homepage documents pool
+    de: Homepage-Dokumente-Becken ))
```

Comment:
not many possibilities here, best would be probably: "Homepage-Dokumente-Fundus"